samples/daily-prebuilt/permissions-can-admin/package-lock.json

```diff
@@ -440,6 +440,21 @@
         }
       }
     },
```

Comment:
Just ran `npm i` as I noticed a diff here post merge from main and it created these package updates